spikesorters/yass/yass.py

```diff
@@ -0,0 +1,280 @@
+import copy
+from pathlib import Path
+import os
+import numpy as np
+from numpy.lib.format import open_memmap
+import sys
+import yaml
+
+import spikeextractors as se
+
+from ..basesorter import BaseSorter
+from ..sorter_tools import recover_recording
+
+from ..utils.shellscript import ShellScript
+
+
+from shellscript import ShellScript
+
+try:
+    import yass
+    HAVE_YASS = True
+except ImportError:
+    HAVE_YASS = False
+
+print ("HAVE_YASS: ", HAVE_YASS) 
+
+#####################################################################
+############# YASSSORTER CLASS ######################################
+#####################################################################
+class YassSorter(BaseSorter):
+    """ 
+    """
+
+    sorter_name = 'yass'
+    requires_locations = False
+    verbose = False
+    is_installed = HAVE_YASS
+
+    # _default_params comes from default Yass config.yaml file; 
+    #  Should be autoloaded from yass_installation_directory/samples/10chan/config.yaml
+    #  - for now a copy is saved here: spikesorters/spikesorters/yass/config.yaml
+    
+#     _default_params = {
+#         'adjacency_radius': 100,  # Channel neighborhood adjacency radius corresponding to geom file
+#         'filter': True,
+#         }
+
+    # ALESSIO: Move thsi file locally to the yass directory;
+    default_config_fname = './config_sample.yaml'
```

Comment:
you should use the `__file__` to be sure the './config_sample.yaml' is working.

Reply:
Not clear what this means. Can you please elabroate